### ISSUE TITLE
Android 10

### ISSUE THREAD

**Issue by liljude** [phhusson/treble_experimentations#1867]
My name is Asea Jude i operate tecno camon 11 pro cf8 mt6765 tried installing treble version of Android 10 but bricked my phone, what can I do to install it

**Comment by liljude** [comment]
![Uploading Screenshot_20210526-073012.png…]()

**Comment by liljude** [comment]
![Screenshot_20210526-072955](https://user-images.githubusercontent.com/50477182/119602694-5de6aa80-bdf4-11eb-80a8-9f1616702a76.png)
Those are my device details

**Comment by phhusson** [comment]
What did you flash? How?

**Comment by liljude** [comment]
Flashed the system.img file using twrp recovery

**Comment by krynyx** [comment]
What is the system version of the device you overlaid the system partition?
Have you deleted the data partition?

**Comment by liljude** [comment]
No, I did not delete data partition, my system version is android 8.1.0
CF8-H622AB-O-201102V274
